docs/tdvt.md

```diff
@@ -544,3 +544,41 @@ Check that any <span style="font-family: courier new">date-format</span> element
 
 __The agg.countd expression test and the join.null.int logical tests are failing__
 Check that your database is correctly returning column nullability information in the metadata. See [Design Considerations]({{ site.baseurl }}/docs/design) for more information.
+
+## Running TDVT test with tabquerytoolsrv.exe
```

Comment:
The focus of this documentation should be the Linux use case. 

I think we want to fit this into the https://tableau.github.io/connector-plugin-sdk/docs/tdvt#set-up part. Windows and Mac users should continue to use the Desktop installers. Linux uses should install server and configure TAB_CLI_EXE_LINUX.

Reply:
Remove the windows part for the documentation.

Reply:
Updated the title to refect this is only for linux server.